pubs/commands/note_cmd.py

```diff
@@ -18,7 +19,12 @@ def command(conf, args):
     ui = get_ui()
     rp = repo.Repository(conf)
     citekey = resolve_citekey(rp, args.citekey, ui=ui, exit_on_fail=True)
+    append = args.append
     notepath = rp.databroker.real_notepath(citekey, rp.conf['main']['note_extension'])
-    ui.edit_file(notepath, temporary=False)
+    if append is None:
+        ui.edit_file(notepath, temporary=False)
+    else:
+        with open(notepath, 'a') as fd:
```

Comment:
With Py27 support still ongoing, and encoding problems galore,  here you should use this function instead:
https://github.com/pubs/pubs/blob/5551e8c04b21fd1ba5a45a204f25d19ab7727462/pubs/content.py#L98